Task0.8.js

```diff
@@ -0,0 +1,12 @@
+function convertToHoursMinutes(number){
+    var display= " ";
+    var hours = Math.floor(number/60);//Uses Math.floor method to determin how many hours are in the number.
+    var minutes  = number % 60;//Uses the Modulus operator to get minutes remaining on the number.
+    //Making use of Ternary operators to control singulars and plurals for hours and minutes.
```

Comment:
Don't comment everything. Only comment when the code is not easy to understand - not when it is self explanatory

Reply:
Thank you Sbonelo. Noted. fixing right away.